### DISCUSSION TITLE
Setting to change default sort for collection docs overview

### DISCUSSION THREAD

**Discussion by JoeyJanson** [outline/outline#1355] [General]
Default is _**Recently updated**_
Would be nice to have a setting to set the default to A-Z

Outline screenshot:
![image](https://user-images.githubusercontent.com/22045972/87539496-48660780-c69e-11ea-9481-ce809d98a15a.png)

**Comment by tommoor** [comment]
Hey @JoeyJanson – agreed! I recently added an issue to track this, the engineering lift should be small - https://github.com/outline/outline/issues/1310

What's your usecase, is there a particular type of content in the collection?

**Reply by JoeyJanson** [reply]
Ha, you already created that. Missed that one.

We have a few collections which have step-by-step docs, like:
1. Getting started with <something>
2. Installing <something>
3. Extra info for <something>

So, having the default sort set to A-Z would already work for me.
